src/main/java/com/resource/TwitterController.java

```diff
@@ -16,13 +19,14 @@
     TweetPostRequest tweetPostRequest;
     SendTweet sendTweet;
     RetrieveTweets retrieveTweets;
+    TwitterImpl twitterimpl=new TwitterImpl();
```

Comment:
Make one more constructor(parameter), for testing 
{
// pass mock from test case
assign parameter
this,object - object 
}
// default constructor
classname () { // normal flow --  constructor
twitterimpl=new TwitterImpl(); 
}